### ISSUE TITLE
[request] different style captcha option 

### ISSUE THREAD

**Issue by 1sad** [K1rakishou/Kuroba-Experimental#465]
sliding one I have a lot of trouble with sometimes (maybe I'm not used to it yet, but sometimes takes a minute) 
this seems like an easier way to me
the option to disable colored background on the sliding one would also be nice (like on 4chan x, seems a little easier to mach up when its same colors)

first screen shows rough lineups, usually easy to spot the correct one side by side
![Screenshot_2021-07-06-23-14-01](https://user-images.githubusercontent.com/23120024/124661834-b9ef2500-deb0-11eb-8b52-2ce66f17162c.png)

second to sweeten the selection
![Screenshot_2021-07-06-23-14-09](https://user-images.githubusercontent.com/23120024/124661960-ddb26b00-deb0-11eb-84bb-9c52237bde67.png)

third is like good old text captcha, almost
![Screenshot_2021-07-06-23-14-13](https://user-images.githubusercontent.com/23120024/124662025-edca4a80-deb0-11eb-8f58-de2c76d93c52.png)

it's from dashchans 4chan plugin update

**Comment by K1rakishou** [comment]
Yeah dashchan's implementation is pretty interesting but I'm not going to adopt it right now because, who knows, the captcha may get changed since it doesn't seem very secure.

**Comment by 1sad** [comment]
thats fair, i hope they figure out a better captcha service, this one is very annoying

how about an option to disable colored background, so its all same color? would be nice to have that option

**Comment by baraa272** [comment]
hi k1 
thanks for implementing the captcha fix quick but it seems there is a problem with it 
the image displayed on portait mode is cutoff but display fine on landscape so can you look into this issue ?
i actually opened a github issue with screenshots provided

**Comment by K1rakishou** [comment]
@baraa272 Yeah I already saw that.

**Comment by baraa272** [comment]
thx dev

**Comment by Mishiranu** [comment]
Actually, the reason it works like this in Dashchan is because its modular design which doesn't allow to do arbitrary stuff without modifying both client and extension. When I came up with this idea, I wasn't sure about it until it was implemented, and only then I said to myself, "wow, it's quite usable".

> first screen, second, …

It's the binary search. The amount of steps depends on the difference between widths of two images, so it can take 3 screens or even more. You can always skip the remaining steps if you think the option you've chosen is good enough by pressing "OK" (even the first step can be skipped like that, but you'll likely get a mess).

**Comment by K1rakishou** [comment]
>Actually, the reason it works like this in Dashchan is because its modular design which doesn't allow to do arbitrary stuff without modifying both client and extension.

Yeah I figured this was the reason. Still it's a pretty untrivial approach, I probably wouldn't came up with something like this.

**Comment by lkd70** [comment]
> Yeah dashchan's implementation is pretty interesting but I'm not going to adopt it right now because, who knows, the captcha may get changed since it doesn't seem very secure.

Certainly not a great captcha, not sure if you've checked this out yet:

https://github.com/AUTOMATIC1111/4chan-captcha-solver

**Comment by kavemang** [comment]
Ive been using this app for about a year and have never been able to post through it, I dont get the captcha at all, no matter where the slider is I cant verify
